file-formats/suco/type/zif_aff_suco_v1.intf.json

```
@@ -0,0 +1,7 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Apllication Default Variants",
```

```suggestion
    "description": "Aplication Default Variants",
```


Is it only 1 variant? You wrote VariantS.
Additionally, what does SUCO stand for? Do you think it would make sense to write any reference to what SUCO stands for?

---

Yes, only one authorization default variant is created or its short text is changed. I changed the texts and inserted one for the whole interface. Now they are more meaningful and singular.

---

@WDFdaniel I think @GuilhermeSaraiva96 was referring to the description of your AFF type's interface. I think at least the typo is still in the code.

---

Oh, sorry! I had only made changes in the interface and did not pay attention to the description in the properties view. I also revised this description now.